apps/design/frontend/src/export_screen.tsx

```diff
@@ -241,7 +215,7 @@ export function ExportScreen(): JSX.Element | null {
             <LoadingButton>Exporting Election Package...</LoadingButton>
           ) : (
             <Button onPress={onPressExportElectionPackage} variant="primary">
-              Export Election Package
+              Export Election & Ballot Packages
```

Comment:
Won't comment on any others since you get the idea haha --> `Export Election Package and Ballots`

Reply:
Thanks, forgot to update those. Also updated tests in [ee381c9](https://github.com/votingworks/vxsuite/pull/5957/commits/ee381c9e5c57a149de4bc24ccf13ff4f7e114b08) but will probably open a separate PR after https://github.com/votingworks/vxsuite/pull/5966 is merged

Reply:
Sounds good!